sdk/ai/azure-ai-inference/README.md

```diff
@@ -44,35 +59,51 @@ pip install --upgrade azure-ai-inference
 
 ## Key concepts
 
-### Create and authenticate a client directly, using key
+### Create and authenticate a client directly, using API key or GitHub token
 
-The package includes two clients `ChatCompletionsClient` and `EmbeddingsClient`<!-- and `ImageGenerationClients`-->. Both can be created in the similar manner. For example, assuming `endpoint` and `key` are strings holding your endpoint URL and key, this Python code will create and authenticate a synchronous `ChatCompletionsClient`:
+The package includes two clients `ChatCompletionsClient` and `EmbeddingsClient`<!-- and `ImageGenerationClients`-->. Both can be created in the similar manner. For example, assuming `endpoint`, `key` and `github_token` are strings holding your endpoint URL, API key or GitHub token, this Python code will create and authenticate a synchronous `ChatCompletionsClient`:
 
 ```python
 from azure.ai.inference import ChatCompletionsClient
 from azure.core.credentials import AzureKeyCredential
 
+# For GitHub models
 client = ChatCompletionsClient(
-    endpoint=endpoint,
+    endpoint="https://models.inference.ai.azure.com",
+    credential=AzureKeyCredential(github_token),
+    model="mistral-large" # Update as needed. Alternatively, you can include this is the `complete` call.
+)
+
+# For Serverless API or Managed Compute endpoints
+client = ChatCompletionsClient(
+    endpoint=endpoint,  # Of the form https://<your-host-name>.<your-azure-region>.models.ai.azure.com
     credential=AzureKeyCredential(key)
 )
+
+# For Azure OpenAI endpoint
+client = ChatCompletionsClient(
+    endpoint=endpoint,  # Of the form https://<your-resouce-name>.openai.azure.com/openai/deployments/<your-deployment-name>
+    credential=AzureKeyCredential("key"),
+    api_version="2024-06-01",  # AOAI api-version. Update as needed.
+)
```

Comment:
Passing key like this won't work for AOAI now, should we remove this example?

Reply:
Oh, good catch! I fixed it to show the same thing we show in the samples. Thanks!